modules/misc/xdg-autostart.nix

```diff
@@ -0,0 +1,24 @@
+{ pkgs, lib, config, ... }:
+
+with lib;
+
+{
+  meta.maintainers = with maintainers; [ jD91mZM2 ];
+
+  options.xdg.autoStart = mkOption {
+    type = types.bool;
+    description = ''
+      Automatically start all desktop items in XDG autostart directories (using dex)
+    '';
+    default = false;
+  };
```

Comment:
You can use mkEnableOption for this

Reply:
True, I just wasn't sure if it counted as an enable option or if I was being lazy. Will fix :)